src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -248,7 +248,7 @@ func getDirectoriesToDelete(keepSystemImages bool, appName string) ([]string, er
 			}
 		}
 	} else {
-		dirs = append(dirs, localRDAppData)
+		dirs = append(dirs, localRDAppData, path.Join(localRDAppData, "cache", "updater-longhorn.json"))
```

Comment:
This seems incorrect. If we're deleting all of `localRDAppData`, we shouldn't need to delete stuff inside of it.

I think you want this in the other branch of `if keepSystemImages`. Except that I think it's covered by existing code already:
https://github.com/rancher-sandbox/rancher-desktop/blob/0400b21c2fadff7fc9ce9083359d2bedc6f1c15a/src/go/rdctl/pkg/factoryreset/factory_reset_windows.go#L242-L243

Reply:
Yes, I didn't even realize that the windows code has a blacklist for specific entries in `cache` not to be deleted. So this change isn't needed.